src/uu/cut/src/cut.rs

```diff
@@ -267,10 +267,45 @@ fn cut_fields_implicit_out_delim<R: Read, M: Matcher>(
     Ok(())
 }
 
+fn cut_fields_newline_char_delim<R: Read>(
```

Comment:
maybe add a comment documenting the function

Reply:
Done.